src/App.tsx

```diff
@@ -10,9 +13,17 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
```

Comment:
Make a separate component for it
```suggestion
            <Route path="/" element={...} />
```